src/Field/Email.php

```diff
@@ -49,6 +49,10 @@ class Email extends Field
      */
     public function normalize($value)
```

Comment:
```suggestion
    public function normalize($value): ?string
```

Reply:
Parent Field class do not have return value type set. Will that be OK to set it here?